client/src/js/services/JournalVoucherService.js

```diff
@@ -0,0 +1,28 @@
+angular.module('bhima.services')
+.service('JournalVoucherService', JournalVoucherService);
+
+JournalVoucherService.$inject = [ '$http', 'util'];
+
+/**
+ * @module JournalVoucherService
+ */
+
+function JournalVoucherService($http, util) {
```

Comment:
There is already a service called `VoucherService` for creating journal vouchers.  You should you that service.